src/analysis/object.rs

```diff
@@ -61,8 +61,11 @@ pub fn class(env: &Env, obj: &GObject, deps: &[library::TypeId]) -> Option<Info>
     let klass: &library::Class = type_.maybe_ref()?;
 
     let mut imports = Imports::with_defined(&env.library, &name);
+    if !env.library.is_glib_crate() {
+        imports.add("glib");
```

Comment:
This shouldn't be needed any more. If `glib` is in the dependencies you can directly use `glib::blabla` everywhere

Reply:
:o